### PR TITLE
adding three-node bare metal TP

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@sheriff-rh, @ahardin-rh, will you PTAL?

File: release_notes/ocp-4-2-release-notes.adoc [hunk 1]
```diff
@@ -115,6 +115,10 @@ xref:../installing/installing_restricted_networks/installing-restricted-networks
 and xref:../operators/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
 for details.
 
+[id="ocp-4-2-three-node-bare-metal-deployments"]
+==== Three-node bare metal deployments (Technology Preview)
+In {product-title} {product-version}, the ability to deploy a cluster without compute, or worker nodes, on bare metal clusters is a technology preview. If you do not create worker machines when you install your cluster on bare metal infrastructure that you provision, all pods, including the router pods, are deployed on the control plane, or master, machines instead. Because it is not recommended to use technology preview features in production clusters, do not use three-node clusters for production. Note that this deployment method is not available in cloud-based clusters in {product-title} {product-version}.
```
Comment (smarterclayton) [inline release_notes/ocp-4-2-release-notes.adoc]: > ... without distinct compute, or worker nodes, on bare metal clusters ...

I think is slightly better

> We do not recommend using technology preview features in production clusters at this time.

maybe?

Comment (kalexand-rh) [reply release_notes/ocp-4-2-release-notes.adoc]: The first is great, but we can't say "we" in the docs. I'm happy with this, though:

> It is not recommended to use technology preview features in production clusters.

File: release_notes/ocp-4-2-release-notes.adoc [hunk 2]
```diff
@@ -115,6 +115,10 @@ xref:../installing/installing_restricted_networks/installing-restricted-networks
 and xref:../operators/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
 for details.
 
+[id="ocp-4-2-three-node-bare-metal-deployments"]
+==== Three-node bare metal deployments (Technology Preview)
+In {product-title} {product-version}, the ability to deploy a cluster without distinct compute, or worker nodes, on bare metal clusters is a technology preview. If you do not create worker machines when you install your cluster on bare metal infrastructure that you provision, all pods, including the router pods, are deployed on the control plane, or master, machines instead. Note that this deployment method is not available in cloud-based clusters in {product-title} {product-version}. It is not recommended to use technology preview features in production clusters.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-2-release-notes.adoc]: s/is a technology preview/a Technology Preview feature 
or
s/is a technology preview/in Technology Preview

We treat TP as a proper noun. You may also want to add an xref to ocp-4-2-technology-preview, which further defines TP and lists all TP features.

s/pods/Pods

**Comment by ahardin-rh** [comment]
Just some minor comments. Looks good!

**Comment by kalexand-rh** [comment]
Thank you @ahardin-rh!

**Pull request merged**